sanic/helpers.py

```diff
@@ -131,3 +135,21 @@ def remove_entity_headers(headers, allowed=("content-location", "expires")):
         if not is_entity_header(header) or header.lower() in allowed
     }
     return headers
+
+
+def import_string(module_name):
+    """
+    import a module or class by string path.
+
+    :module_name: str with path of module or path to import and
+    instanciate a class
+    :returns: a module object or one instance from class if
+    module_name is a valid path to class
+
+    """
+    module, klass = module_name.rsplit(".", 1)
```

Comment:
I might be overthinking this, but would it be a good idea to support relative string paths?

Reply:
well, i use this function in Flask for configure app using config_from_object and is very good.
Because is common in application has config classes.
example:
```
class Config:
      DEBUG = False

class Develop(Config):
    DEBUG = True

class Production(Config):
      pass
```

Reply:
> In this case the user instanciate Sanic class and configure according from environment pass the string path of the class get from environment variable for example.

Oh I am totally with you on this feature. I was just curious if it might be possible to support relative path as well. 

i.e. Instead of `module1.module2.Class`, if you are already inside `module1`, then `module2.Class1` should suffice.

Reply:
@vltr It works, but the method invocation might need to be tweaked a bit to provide another helper argument. `import_module(name, package=None)`

Reply:
@jotagesales looks like this is still outstanding - can you make changes so we can get this in soon?

Reply:
@sjsadowski ready ;)